src/App.js

```diff
@@ -1,5 +1,25 @@
+import { parsePlayers, parseRound } from './parseInput';
+import { processInputToOutput } from './Simulate';
+import { formatOutput } from './formatOutput';
+
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Console = MissionUtils.Console;
+
+const Message = {
+  playerList:
+    '경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)\n',
+  roundNumber: '시도할 횟수는 몇 회인가요?\n',
+};
+
 class App {
-  async run() {}
+  async run() {
+    const inputList = await Console.readLineAsync(Message.playerList);
+    let players = parsePlayers(inputList);
+    const inputRound = await Console.readLineAsync(Message.roundNumber);
+    let round = parseRound(inputRound);
```

Comment:
```suggestion
    const players = parsePlayers(inputList);
    const inputRound = await Console.readLineAsync(Message.roundNumber);
    const round = parseRound(inputRound);
```
여기서 `players`, `round`는 재할당되는 로직이 없는 것 같은데 `const` 키워드로 작성하는건 어떨까요??